src/main/java/software/bernie/techarium/pipes/networks/ItemFilter.java

```diff
@@ -0,0 +1,24 @@
+package software.bernie.techarium.pipes.networks;
+
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+
+public class ItemFilter extends Filter<ItemStack> {
+    private Item[] filtered = {Items.AIR,Items.AIR,Items.AIR,Items.AIR,Items.AIR};
```

Comment:
I dunno if it's possible but it'd be nice to use `Ingredient` instead so we can use tags and other stuff

Reply:
I'm not sure how you would insert the Tag into the filter in the gui? I thought putting items into fake slots would be easiest, but I can change it, if you come up with a way to display them